src/main/java/org/aksw/simba/lemming/ColouredGraphWrapper.java

```diff
@@ -0,0 +1,98 @@
+package org.aksw.simba.lemming;
+
+import java.io.Serializable;
+
+import org.aksw.simba.lemming.colour.ColourPaletteWrapper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.BitSet;
+import java.util.List;
+
+import grph.Grph;
+
+/**
+ * Wrapper Class to enable the persistence of the ColouredGraph class
+ * @author Ana
+ *
+ */
+public class ColouredGraphWrapper implements Serializable {
```

Comment:
I do not understand why this class is necessary. Wouldn't it be sufficient to make the ColouredGraph class implement the Serializable interface? :thinking:

Reply:
At the time, I found several usages of BitSet, ObjectObjectOpenHashMap, ObjectArrayList from com.carrotsearch.hppc which do not implement Serializable. I created these Wrapper classes to use java.util classes, analogous to the hppc ones, thus allowing serialization. :)